coord/src/module.c

```diff
@@ -750,6 +751,8 @@ searchResult *newResult(searchResult *cached, MRReply *arr, int j, searchReplyOf
       if (eptr != res->sortKey + 1 && *eptr == 0) {
         res->sortKeyNum = d;
       }
+    } else if (!strncmp(res->sortKey, "none", 4)) {
+      res->sortKey = NULL;
```

Comment:
Do we need to free `res->sortKey`?

Reply:
No. We do not take ownership of the MRReply elements.

Reply:
So who will free it?

Reply:
`MRCtx_Free` calls `MRReply_Free` on all the replies.

Reply:
Did you run it with valgrind? its on coordinator and we do not have sanatizer so lets be sure...

Reply:
Ran sanitizer locally and confirmed no memory leak